### PR TITLE
Add support for grid property

### PR DESCRIPTION
**Pull request by paulyoung**
This adds support for a `grid` property, much like the one in `react-draggable`.

Please review 🙏

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -128,24 +131,28 @@ export default class Resizable extends Component {
...
       newWidth = clamp(newWidth, min, max);
+      newWidth = snap(newWidth, this.props.grid[0]);
     }
     if (/bottom/i.test(direction)) {
       newHeight = original.height + clientY - original.y;
       const min = (minHeight < 0 || typeof minHeight === 'undefined') ? 0 : minHeight;
       const max = (maxHeight < 0 || typeof maxHeight === 'undefined') ? newHeight : maxHeight;
       newHeight = clamp(newHeight, min, max);
+      newWidth = snap(newWidth, this.props.grid[1]);
     }
```
Comment (bokuweb) [inline src/index.js]: That is a mistake.

correct?
`newHeight = snap(newHeight, this.props.grid[1]);`

File: src/index.js [hunk 2]
```diff
@@ -128,24 +131,28 @@ export default class Resizable extends Component {
...
     if (/top/i.test(direction)) {
       newHeight = original.height - clientY + original.y;
       const min = (minHeight < 0 || typeof minHeight === 'undefined') ? 0 : minHeight;
       const max = (maxHeight < 0 || typeof maxHeight === 'undefined') ? newHeight : maxHeight;
       newHeight = clamp(newHeight, min, max);
+      newWidth = snap(newWidth, this.props.grid[1]);
```
Comment (bokuweb) [inline src/index.js]: That is a mistake too.

**Comment by bokuweb** [comment]
@paulyoung Hi!! it is a cool request!!! thanks!!
pleease fix mistake, calculate new height, and please add readme about `grid` props🙏

**Comment by paulyoung** [comment]
@bokuweb mistakes fixed and README updated.

**Comment by bokuweb** [comment]
@paulyoung It looks great to me!!!! thanks!!!!!

**Comment by bokuweb** [comment]
v1.4.0 released.

**Comment by paulyoung** [comment]
Thanks @bokuweb!

**Pull request merged**